### ISSUE TITLE
Strange logo resizing issues for the Splash Screen logo

### ISSUE THREAD

**Issue by GwynethLlewelyn** [Myopengrid/mwi#4]
When adding a logo for the Splash Screen, it seems to get resized always to the same size (340x240), no matter what. And this is unfortunate since it will get covered by the Message Block (when active), which is also just 300 px wide, so it won't be properly centred.

Obviously logos can be tweaked in Photoshop to get them to adapt to the template (leaving empty, transparent space in the zones that will be overlapped), but it would be nicer if:

a) The image uploader page informs users that the resulting logo will be resized to 340x240
or b) The logo does not get resized at all

**Comment by Myopengrid** [comment]
Yes this is a "issue" with the layout itself. The size of the logo is hard-coded in the page template, you can change that on the file bundles/splashscreen/views/splash_screen.blade.php on line 21

This "issue" probably will gain some attention in the future.

Thank you.

**Comment by GwynethLlewelyn** [comment]
Ok, I've tested it on CoolVLViewer on a different computer, and the results are, indeed, different: bigger screen sizes will mean more space for the logo, and the overlapping only happens on screen heights under 900px or so.
